shared/src/api/interceptors.coffee

```diff
@@ -0,0 +1,134 @@
+_ = require 'lodash'
+minimatch = require 'minimatch'
+{Promise} = require 'es6-promise'
+
+makeLocalRequest = (requestConfig) ->
+  {url} = requestConfig
+
+  [uri, params] = url.split('?')
+  if requestConfig.method is 'GET'
+    url_parts = ["#{uri}.json", params]
+  else
+    url_parts = ["#{uri}/#{requestConfig.method}.json", params]
+    requestConfig.mockMethod = requestConfig.method
+    requestConfig.method = 'GET'
+
+  requestConfig.url = _.compact(url_parts).join('?')
+
+makeLocalResponse = (response) ->
+  payload = if response.config.data then JSON.parse(response.config.data) else {}
+  response.data = _.extend({}, response.data, payload)
+
+doesErrorMatch = (handledErrors, errorName) ->
+  _.indexOf(handledErrors, errorName) > -1 or
+    _.some(handledErrors, _.partial(minimatch, errorName))
+
+areAllErrorsHandled = (handledErrors, errors, errorNameProperty) ->
+  isErrorHandled = _.partial(doesErrorMatch, handledErrors)
+  _.every errors, _.flow(_.property(errorNameProperty), isErrorHandled)
+
```

Comment:
Maybe GitHub isn't displaying this line correctly, but it looks like you're assigning `isErrorHandled` to y the function returned b`_.partial` and then calling `_.every` on the next line, so it doesn't have any effect.  I suspect the partial should be immediately evaluated?  i.e.:
```
 isErrorHandled = _.partial(doesErrorMatch, handledErrors)(
      _.every errors, _.flow(_.property(errorNameProperty), isErrorHandled)
)
```